internal/storage/local_chunk_manager.go

```diff
@@ -70,13 +79,17 @@ func (lcm *LocalChunkManager) Reader(filePath string) (FileReader, error) {
 func (lcm *LocalChunkManager) Write(filePath string, content []byte) error {
 	absPath := path.Join(lcm.localPath, filePath)
 	dir := path.Dir(absPath)
-	if _, err := os.Stat(dir); os.IsNotExist(err) {
+	exist, err := lcm.Exist(dir)
+	if err != nil {
+		return err
+	}
+	if !exist {
 		err := os.MkdirAll(dir, os.ModePerm)
 		if err != nil {
 			return err
 		}
 	}
-	err := ioutil.WriteFile(absPath, content, os.ModePerm)
+	err = ioutil.WriteFile(absPath, content, os.ModePerm)
```

Comment:
It seems that we can directly return `ioutil.WriteFile(absPath, content, os.ModePerm)`